numpy/random/mtrand/mtrand.pyx

```diff
@@ -4888,6 +4889,247 @@ cdef class RandomState:
 
         return multin
 
+    def _multivariate_hypergeometric1(self, ndarray colors,
```

Comment:
They don't need to be methods either, could just make them stand alone functions as long as they don't have internal state.